src/Avalonia.Visuals/Rendering/SceneGraph/DeferredDrawingContextImpl.cs

```diff
@@ -114,13 +116,13 @@ public void DrawGeometry(IBrush brush, Pen pen, IGeometryImpl geometry)
         }
 
         /// <inheritdoc/>
-        public void DrawImage(IRef<IBitmapImpl> source, double opacity, Rect sourceRect, Rect destRect)
+        public void DrawImage(IRef<IBitmapImpl> source, double opacity, Rect sourceRect, Rect destRect, BitmapScalingMode scalingMode = BitmapScalingMode.LowQuality)
```

Comment:
The [above code](https://github.com/AvaloniaUI/Avalonia/pull/1762/files#diff-0460aa39b2131b54b827f8d2cf63ef0aR36) that you write the default but in this one you write the `BitmapScalingMode scalingMode = BitmapScalingMode.LowQuality` and I think `BitmapScalingMode scalingMode = default` is well for you can fast to change the default code.